pages/docs/getting-started/security.mdx

```diff
@@ -0,0 +1,37 @@
+import { Callout } from "nextra-theme-docs";
+
+# Security and Compliance
+
+We take security and compliance seriously at Glean. Our software engineering team is full-stack and has experience in regulated fields including healthcare infrastructure, and our VP of Technology serves the role of Chief Information Security Officer for oversight purposes. All members of the engineering team undergo annual information security and secure coding training.
```

Comment:
I don't think this first paragraph is necessary, it calls attention to the fact that we don't have a dedicated CISO. (This language was written in response to a question about whether we have a separate security team.)